### PR TITLE
updated what's new, status page and announcements

### PR DESCRIPTION
**Pull request by SenthilKumarSivakami**
1. Included Cloudflare status on the SEED service status page
2. Included an announcement for egress IP trial removal
3. updated what's new for the announcements.

### CODE REVIEW

File: announcements.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 | Date  | Announcement |
 | ------------- |:-------------:|
+| 02 September 2022 | Cloudflare will be discontinuing its private egress IP trial from 08 September 2022. SEED users will not be affected. |
```
Comment (samloh84) [inline announcements.md:5]: Can we amend this to: GCC 2.0 users are advised to not to configure any network policies that use the egress IP, as per our advice to SEED inquiries about the egress IP.

File: seed-status.md [hunk 2]
```diff
@@ -22,3 +22,7 @@ None
 | Date| 21 June 2022|
 | ------------- |:-------------|
 |**Issue summary** | An issue has been identified with Cloudflare services on June 21, 2022  14:34 SGT. A fix has been implemented on June 21, 15:20 SGT and we are monitoring the outcome.<br><br>**Customer Impact:**<br> Users may have observed 500 errors while accessing the Internet and Cloudflare protected SGTS services using Cloudflare WARP.<br>While we monitor the fix, if you are still unable to access the Internet, please turn off WARP to access the Internet and create an [incident support request](raise-an-incident-support-request) with the respective SGTS service or product team.<br>You may also visit the [Cloudflare status page](https://www.Cloudflarestatus.com/) for recent updates on this issue.|
+
+### Cloudflare status
+
+You may also visit the [Cloudflare status page](https://www.Cloudflarestatus.com/) for recent updates on any Cloudflare issues.
```
Comment (samloh84) [inline seed-status.md:28]: You may also visit the ... for recent updates on any issues with Cloudflare Zero Trust, WARP, Cloudflare Tunnel and Cloudflare Access.

**Review by samloh84 — CHANGES_REQUESTED** [review]
I've left a few suggested amendments.

File: announcements.md (hunk 1 above)
Comment (SenthilKumarSivakami) [reply announcements.md:5]: noted. 

 "GCC 2.0 users are advised not to configure any network policies that use the egress IP."
 
 But wont the GCC users be using other egress IP excluding this egrees IP trial ?

File: seed-status.md (hunk 2 above)
Comment (SenthilKumarSivakami) [reply seed-status.md:28]: got it! Thanks :-)

File: announcements.md (hunk 1 above)
Comment (samloh84) [reply announcements.md:5]: The intent is to tell GCC 2.0 users that if they have configured any network policies that use the egress IP that cloudflare is going to take away, they should remove the network policy or prepare to change it.

Comment (SenthilKumarSivakami) [reply announcements.md:5]: ok, i will tweak the content slightly.

Comment (SenthilKumarSivakami) [reply announcements.md]: done.

**Review by samloh84 — APPROVED** [review]
LGTM!

**Review by samloh84 — APPROVED** [review]
LGTM 2nd time!

**Pull request merged**